nubis/Puppetfile

```diff
@@ -54,6 +54,10 @@ mod 'nubis/nubis_storage',
     :git => 'https://github.com/Nubisproject/nubis-puppet-storage.git',
     :ref => 'v1.0.0'
 
+mod 'nubis/nubis_mig',
```

Comment:
This will make the puppet class named 'nubis_mig', even though the puppet code expects it to be called 'mig'